### ISSUE TITLE
[Accessibility] MAS 4.2.1: (eclipse/edit '') The Name property of a focusable elements must not be null.

### ISSUE THREAD

**Issue by soumyatippapuram** [microsoft/azure-tools-for-java#4284]
The following accessibility issue needs investigation.

[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse-Benchmark; #MAS4.2.1; #A11yAuto; #JavatoolingforIntelliJ/Eclipse; #C+E-H; #Benchmark; #A11ySev2; #AI4D  

**NOTE:** Same issue is observed in below screens also
File > New > Project > Select a Wizard
File > New > Project > Select a Wizard > Create a Scala Project
 
**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2020-03 (4.15.0)
Build id: 20200313-1211

**Solution Installed:** The Azure Toolkit for Eclipse 3.220, Scala IDE 4.7.x

**OS Details:**
OS: Windows (Version 1903, Build: 18362.175)

**MAS Violated:** MAS 4.2.1

**App:** eclipse

**Element path:** edit '' , dialog ''

**Issue Details:** The Name property of a focusable element must not be null. [Section 508 502.3.1](https://www.access-board.gov/guidelines-and-standards/communications-and-it/about-the-ict-refresh/final-rule/single-file-version#502-interoperability-assistive-technology)

**How To Fix:** Provide a UI Automation Name property that concisely identifies the element.

This accessibility issue was found using Accessibility Insights for Windows, a tool that helps debug and find accessibility issues earlier. Get more information and download this tool at https://aka.ms/AccessibilityInsights.

**MAS Reference:**
[MAS 4.2.1](https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7BEEE97BBD-9990-48A0-8B03-6EA775581AA8%7D&file=MAS%204.2.1%20%E2%80%93%20Object%20Information.docx&action=default&mobileredirect=true)

![Bug_4284_Java Tooling_Azure Sign in_AI](https://user-images.githubusercontent.com/41108985/78989610-b7890880-7b51-11ea-930b-839aaf644302.png)

![Bug_4284_Java Tooling_Select a Wizard_AI4D](https://user-images.githubusercontent.com/41108985/79008764-b7056780-7b7b-11ea-82e7-571e1b8841c1.png)

![Bug_4284_Java Tooling_Create Scala Project_AI4D](https://user-images.githubusercontent.com/41108985/79009151-82de7680-7b7c-11ea-80e8-688f99ff306d.png)

**Comment by Flanker32** [comment]
@soumyatippapuram Check the codes, the first dialog is extended from class `org.eclipse.jface.dialogs.TitleAreaDialog`, which is provided by Eclipse., and the rest dialogs are provided by Eclipse, both of them are out of the scope of our toolkits. Here is the related [issue](https://github.com/microsoft/azure-tools-for-java/issues/3611),  you may find more infos [here](https://github.com/microsoft/azure-tools-for-java/issues/3611#issuecomment-551441705)

**Comment by Flanker32** [comment]
- Importance of experience: Minimal, when user focus to sub-title textfield or help button, low vision user doesn't know the name of current focused element.
- Quality of experience: Not impacted. The text of the component itself will tell user the meaning of current focused element
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue.
- [AAT Link](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076)

**Comment by Flanker32** [comment]
@soumyatippapuram This issue has been linked to AAT, please have a review, thanks!

**Comment by soumyatippapuram** [comment]
Closing this bug as a similar [issue](https://github.com/microsoft/azure-tools-for-java/issues/3611) was discussed and approved by SME.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749659
